<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QWidget" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="allBox">
       <property name="text">
        <string>Show All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="unpaidBox">
       <property name="text">
        <string>Show UnPaid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="paidBox">
       <property name="text">
        <string>Show Paid</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="companySearch"/>
     </item>
     <item>
      <widget class="QPushButton" name="getTotalButton">
       <property name="text">
        <string>Compute Bills</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Total Unpaid Bills Sum:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="totalSumLabel">
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
